apps/VARA HF/install

```diff
@@ -53,7 +49,7 @@ echo "[Desktop Entry]
 Name=VARA HF
 GenericName=VARA HF
 Comment=VARA HF is a shareware ham radio OFDM software modem for RMS Express and other messaging clients.
-Exec=env WINEDEBUG=-all wine \"C:\\VARA\\VARA.exe\"
+Exec=env WINEDEBUG=-all wine "${PREFIXDIR}"/drive_c/VARA/VARA.exe
```

Comment:
The way these double-quotes are escaped in the echo command, they will not show up in the .desktop file.
They will be necessary if the user's wineprefix contains whitespace characters. I don't recall if single-quote characters work, but to escape the double-quotes, replace them with this: `"\""`